src/Domain/CoordinateMaps/TimeDependent/ShapeMapTransitionFunctions/ShapeMapTransitionFunction.hpp

```diff
@@ -89,15 +89,17 @@ class ShapeMapTransitionFunction : public PUP::able {
       double distorted_radius) const = 0;
   /*!
    * Evaluate the transition function at the Cartesian coordinates divided by
-   * the radius. All divisions over the radius which could potentially be zero
-   * divisions are left to the transition function which uses its knowledge
-   * of the domain to do this safely or throw an appropriate error.
+   * the radius to a power. All divisions over the radius which could
+   * potentially be zero divisions are left to the transition function which
+   * uses its knowledge of the domain to do this safely or throw an appropriate
+   * error.
```

Comment:
Yes, there is one place in the shape map where dividing by r could be taken care of, but my best guess as to why the transition function is responsible for checking the `1/r` is because the transition function could be anything and so it may also need to account for a `1/r` somewhere itself. And, I think, because the only place that `1/r` shows up is with the transition function itself, it made sense to just have the transition function do all of the checks.

Both the `Wedge` and `Sphere` transition functions do check the magnitude of the point (in debug mode) before dividing by it. So these checks do exist.

Reply:
You're right that I missed the checks in the transition functions, but why not move them to the shape map itself to avoid duplication?  The shape map isn't valid at r=0, so there can't be any issues with dividing by r.

Reply:
Ok yeah, I'm convinced this should be in the shape map now. If the transition function itself needs to divide by r it can just assume it's not zero since the shape map took care of it.

Reply:
If we want to consider using the map in the opposite sense, to remove distortion when going inwards, then you could get a valid map at the origin where I think you might want to remove cancellations in the transition function, but I still don't think you'd want to do it with varying powers in the fetch functions.  You'd want the interface to only provide the valid quantities.